src/main/java/org/squiddev/plethora/integration/appliedenergistics/MetaAppliedEnergistics.java

```diff
@@ -67,6 +69,8 @@ public IAEItemStack getExample() {
 		}
 	};
 
+	private static final MetaFluidStack META_FLUID_STACK = new MetaFluidStack();
```

Comment:
Can you refactor the `MetaFluidStack.getMeta` into a static method (which the instance method just delegates too)? Just saves us having to instantiate it here.